skiko/src/awtMain/kotlin/org/jetbrains/skiko/SystemTheme.awt.kt

```diff
@@ -1,11 +1,11 @@
 package org.jetbrains.skiko
```

Comment:
FYI the file name change is causing a `java.lang.UnsatisfiedLinkError`. See issue https://github.com/JetBrains/compose-jb/issues/1761. There is a PR to fix it here: https://github.com/JetBrains/skiko/pull/464.